scripts/sessionx.sh

```diff
@@ -39,6 +39,7 @@ handle_binds() {
   bind_window_mode=$(tmux_option_or_fallback "@sessionx-bind-window-mode" "ctrl-w")
   bind_configuration_mode=$(tmux_option_or_fallback "@sessionx-bind-configuration-path" "ctrl-x")
   bind_rename_session=$(tmux_option_or_fallback "@sessionx-bind-rename-session" "ctrl-r")
+	additional_fzf_options=$(tmux_option_or_fallback "@sessionx-fzf-options" "--color pointer:9,spinner:92,marker:46")
```

Comment:
## Default Option

This default option: `--color pointer:9,spinner:92,marker:46` is the default behavior?

Maybe instead of leaving an option already configured, we could leave it empty, what do you think?

Reply:
For me, this option sets another color:
![image](https://github.com/omerxx/tmux-sessionx/assets/108758883/0a1817a0-020d-440c-916c-1be87068c184)

Before these changes:
![image](https://github.com/omerxx/tmux-sessionx/assets/108758883/7dd7f82f-e7be-4889-9bfb-4d4154cd1bbc)

For me ok, but I don't know about the other users

Reply:
That's strange, since they are the same values. Maybe something to do with the order in which they are applied? Previously they were set in the middle of the list of args, and now they are applied first.

Ultimately I will defer to the preferences of you and other maintainers on what the default value should be for this option. Are you leaning toward leaving it empty by default?